src/pages/Profile.jsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { useSelector } from "react-redux";
```

Comment:
- [ ] Your profile component does not render, it only displays a blank screen. Note that your profile page should display the list of reservations you have made. Kindly fix this, please.
### Your design
![2](https://user-images.githubusercontent.com/94551484/204074804-30221b6d-0bd6-46e7-a563-705214553b7e.png)


### Guideline
![image](https://user-images.githubusercontent.com/94551484/204074788-695fb8dd-0b72-4ae3-a3bd-b5502d9d863b.png)